week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,6 +1,6 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
```

Comment:
Good. Normally we put the space after the comma like this:
```suggestion
function addNumbers(a, b, c) {
```
(but that is just for neatness)

Reply:
so it's just like writing in English. Sure I will do that